app/tests/spec/models/password_strength/password_strength_balloon.js

```diff
@@ -44,6 +44,7 @@ describe('models/password_strength/password_strength_balloon', () => {
       'asdftesTusEr@Testuser.Com!@%$', // contains the full email
       'TESTUSER!@%%ASDF',              // local part at the beginning
       'TESTUSER@testuser',             // substring of email at the beginning
+      '123TESTUSER',                   // local part comprises over 50% of password
```

Comment:
If the 50% is significant, should we be asserting one character either side of the 50% boundary?

Reply:
> should we be asserting one character either side of the 50% boundary?

Yes, yes we should.